x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_exceptions_query.ts

```diff
@@ -41,37 +47,32 @@ export const operatorBuilder = ({
   operator,
   language,
 }: {
-  operator: ListOperator;
+  operator: Operator;
   language: Language;
 }): string => {
-  const and = getLanguageBooleanOperator({
-    language,
-    value: 'and',
-  });
-  const or = getLanguageBooleanOperator({
+  const not = getLanguageBooleanOperator({
     language,
     value: 'not',
   });
 
   switch (operator) {
-    case 'excluded':
-      return ` ${and} `;
+    // case 'excluded':
+    //   return ` ${and} `;
```

Comment:
not sure if these are supposed to be commented out?

Reply:
Ah, thanks. Yea, they are meant to, just needed to delete them.